packages/frontend/components/Sidebar/SectionRequirement.tsx

```diff
@@ -47,6 +49,23 @@ const SectionRequirement: React.FC<SidebarRequirementProps> = ({
     }
   };
 
+  const isCourseInPlan = (requirement: Requirement2): boolean => {
+    let isTrue = false;
+    if (coursesTaken) {
+      coursesTaken.forEach((course) => {
+        if (
+          requirement.type === "COURSE" &&
+          course.classId === requirement.classId.toString() &&
```

Comment:
can you look into using the `getCourseDisplayString()`? would that work in this situation?